packages/chat-stateful-client/src/iterators/createDecoratedListReadReceipts.ts

```diff
@@ -14,6 +14,7 @@ export const createDecoratedListReadReceipts = (
   context: ChatContext
 ): ((options?: RestListReadReceiptsOptions | undefined) => PagedAsyncIterableIterator<ChatMessageReadReceipt>) => {
   const setReadReceipt = (readReceipt: ChatMessageReadReceipt, context: ChatContext): void => {
+    //check here
```

Comment:
I find some of these comments arent really descriptive and they are creating more questions than anything. Lets make sure that if we write a comment it creates value for anyone reading it.

Reply:
Sorry I left some comments for myself to debug, forgot to remove them, I will go through the changes again to make sure no debugging comments are left